src/Symfony/Component/Workflow/Dumper/GraphvizDumper.php

```diff
@@ -84,7 +84,7 @@ private function findTransitions(Definition $definition)
         $transitions = array();
 
         foreach ($definition->getTransitions() as $transition) {
-            $transitions[] = array(
+            $transitions[$transition->getName()] = array(
```

Comment:
looks wrong to me. Transition names are not unique in the definition (the only requirement is that transition starting from a given node have unique names)